docs/whats-new.asciidoc

```diff
@@ -2,6 +2,28 @@
 [chapter]
 = What's new
 
+[discrete]
+[[sec-7.11-release]]
+== 7.11 release
+
+[discrete]
+[[sec-enhancements-7.11]]
+=== New features
+
+* New <<prebuilt-rules, prebuilt detection rules>>.
+* <<detections-ui-exceptions, Exceptions>> can be assigned to to threshold and {ml} rules.
+* Enhanced UI to <<manage-exceptions, view and manage exceptions>>.
+* Add MITRE ATT&CK sub-techniques in <<rule-ui-advanced-params, advanced rule settings>>.
+* New <<rule-notifications, rule actions>> and enhanced <<rule-action-variables, alert notifications>>.
+* New support for <<cold-tier-detections, cold tier data>> and searchable snapshots for specific {es} indices.
+* <<self-protection, Self-protection>> enabled on Windows and macOS by default.
+* Register {es-sec} as an <<register-as-antivirus, antivirus solution>> on Windows.
+* Customize <<malware-protection, malware>> notification messages.
+* Enhanced <<timelines-ui, Timeline>> design with accessibility features.
+* Enhanced capability to add a <<trusted-apps-ov, trusted application>> by signer.
+* Enhanced <<visual-event-analyzer, event visualization>> for Endpoint and Windows process events.
+* New <<signals-migration-api, detection alerts migration API>> feature, which can be used to enable new features on existing detection alerts.
+
```

Comment:
Similar to the wording used for new ML jobs in 7.10, I'd suggest adding something like this:
```suggestion
* Fourteen new {ml} {anomaly-jobs} have been added, which support multi-index analysis for Linux or Windows data and detect anomalous user, process, and network port activity. See <<security-linux-jobs>> and <<security-windows-jobs>>.

```